src/types/interfaces/estimate.ts

```
@@ -0,0 +1,191 @@
+	TxnType: string;
+	TxnLineId?: string;
+}
+
+/**
+ * The EstimateLine Object
+ */
+interface EstimateLine {
+	readonly Id?: string;
+	DetailType: LineDetailTypeEnum;
+	SalesItemLineDetail?: SalesItemLineDetail;
+	SubTotalLineDetail?: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)